box-content-sdk/src/main/java/com/box/androidsdk/content/requests/BoxRequestsFile.java

```diff
@@ -1206,4 +1221,379 @@ public void setRequestedPage(int page) {
             mRequestPage = page;
         }
     }
+    /**
+     * Request for creating an upload session for multiput upload
+     */
+    public static class CreateUploadSession extends BoxRequest<BoxUploadSession, CreateUploadSession> {
+        private static final long serialVersionUID = 8145675031279971502L;
+
+        String mDestinationFolderId;
```

Comment:
Add "protected" for these other member variables so children classes from different packages can read them and easily extend